wordview/text_analysis/wrapper.py

```diff
@@ -242,9 +310,9 @@ def show_stats(self) -> None:
                 ["All Words", f"{self.token_count:,d}"],
                 ["Documents", f"{self.num_docs:,d}"],
                 ["Median Doc Length", self.median_doc_len],
-                ["Nouns", f"{self.num_nns:,d}"],
-                ["Adjectives", f"{self.num_jjs:,d}"],
-                ["Verbs", f"{self.num_vbs:,d}"],
+                ["Nouns", f"{self.pos_counts['NN']:,d}"],
+                ["Adjectives", f"{self.pos_counts['JJ']:,d}"],
+                ["Verbs", f"{self.pos_counts['VB']:,d}"],
             ],
```

Comment:
Now that we have more tags, thanks to you, let's also add NNP and RB to this table, which I think are pretty important tags.

Reply:
Thanks for pointing out those issues, especially the problem with the variable names. I have fixed them in the latest commit.